scripts/generate_tags.py

```diff
@@ -7,8 +7,51 @@
 import re
 from urllib.request import urlopen
 
+_INLINE_TAG_NAMES = {
```

Comment:
Looks like you got this list from #34, but I don't remember where I got that list (sorry), and this seems that there are more tags to consider adding here https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements#list_of_inline_elements